src/amo/components/AddonReviewCard/index.js

```diff
@@ -423,6 +423,11 @@ export class AddonReviewCardBase extends React.Component<InternalProps> {
       (review.userId === siteUser.id ||
         (this.isReply() && this.siteUserCanManageReplies()));
 
+    /* Do not render the "flag" button for reviews made by the current user */
```

Comment:
Prefix comments with `//` (double slashes) when they don't need to span multiple lines.